tests/v1/engine/test_engine_core_client.py

```diff
@@ -293,7 +293,7 @@ def test_kv_cache_events(
         subscriber = MockSubscriber(endpoint,
                                     topic=publisher_config.topic,
                                     decode_type=KVEventBatch)
-
+        time.sleep(1)  # Wait for subscriber to connect
```

Comment:
In ZeroMQ, after a subscriber sets a subscription, it takes some time for the subscription request to reach the publisher. Therefore, if the publisher sends messages immediately after starting, those messages may not be received by the subscriber. To ensure message delivery, it is recommended to add a short delay before the publisher starts sending messages.